src/robusta/core/playbooks/internal/ai_integration.py

```diff
@@ -454,11 +541,79 @@ def holmes_workload_chat(event: ExecutionBaseEvent, params: HolmesWorkloadHealth
             failure=False,
         )
         finding.add_enrichment(
-            [HolmesChatResultsBlock(holmes_result=holmes_result)], enrichment_type=EnrichmentType.ai_analysis
+            [HolmesChatResultsBlock(holmes_result=holmes_result)],
+            enrichment_type=EnrichmentType.ai_analysis,
         )
 
         event.add_finding(finding)
 
     except Exception as e:
-        logging.exception(f"Failed to get holmes chat for health check of {params.resource}", exc_info=True)
+        logging.exception(
+            f"Failed to get holmes chat for health check of {params.resource}",
+            exc_info=True,
+        )
         handle_holmes_error(e)
+
+
+def stream_and_render_graphs(url, holmes_req, event):
+    with requests.post(
+        url,
+        data=holmes_req.json(),
+        stream=True,
+        headers={"Connection": "keep-alive"},
+    ) as resp:
+        resp.raise_for_status()
+        for stream_event in resp.iter_content(
+            chunk_size=None, decode_unicode=True
+        ):  # Avoid streaming chunks from holmes. send them as they arrive.
+            if stream_event:
+                event_lines = stream_event.splitlines()
+                event_type = parse_sse_event_type(event_lines[0])
+
+                if event_type == StreamEvents.ANSWER_END.value:
+                    stream_event = re.sub(r"<<.*?>>", "", stream_event)
```

Comment:
add comment that this strips all embedded tool calls so future people will be aware of the magic